articles/connections/references/apple-native/guides/add-siwa-to-native-app.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Add Sign In with Apple to Your Native App
+description: Learn how to add native login functionality to your native app with Apple. 
+toc: true
+topics:
+  - authentication
+  - connections
+  - social
+  - apple
+contentType: how-to
+useCase:
+  - add-login
+  - customize-connections
+  - add-idp
+---
+# Add Sign In with Apple to Your Native App
+
+To set up and configure Sign In with Apple for your native app, you will do the following:
+
+1. [Complete the prerequisites](#before-you-begin): Have an Apple Developer account (paid), an Auth0 tenant set up with a custom domain, and a web application configured to use Auth0 for authentication at that domain.  
```

Comment:
custom domains are not an Auth0 tenant requirement.  Non-custom domains would work, so long as your email relays are configured appropriately (if you even need those).